cookiecutter.json

```diff
@@ -13,5 +13,7 @@
     "use_celery": "n",
     "use_maildump": "n",
     "use_sentry": "n",
+    "use_docker": "n",
```

Comment:
> People will end up with all the docker specific files anyway because there is no way to exclude files.

but I exclude them :wink: 

Please read description of PR carefully:

> if user decide not to use pycharm or docker, proper files are removed in post_gen_project.py hook

----

> For the time being, we should make sure that we keep these side by side without introducing too much switches.

This is a though topic. I believe, that user should use defaults answers (for most feautes right now, default is `n`), and only type `y`, when knows what exactly this mean. Moreover I want to keep final product of generation as clean as possible, and those docker-related files, IMO actually makes a difference:

![with and without docker](http://i.imgur.com/43x6YMI.png)

@pydanny what do you think?